ingests_tracker/src/main/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApi.scala

```diff
@@ -27,9 +35,19 @@ trait IngestsTrackerApi extends Runnable with Logging {
     concat(
       get {
         pathPrefix("ingest" / JavaUUID) { id =>
-          // val result: ingestTracker.Result = ingestTracker.get(IngestID(id))
-
-          complete(StatusCodes.NotFound)
+          ingestTracker.get(IngestID(id)) match {
+            case Left(IngestDoesNotExistError(_)) =>
+              complete(StatusCodes.NotFound)
+            case Left(_) =>
+              complete(StatusCodes.InternalServerError)
```

Comment:
I can't remember – does this error get logged elsewhere?

Reply:
No, it should.

👍 is in next PR.